### DISCUSSION TITLE
How to use 1 Minute CSV Historical Data from Metatrader for Backtesting?

### DISCUSSION THREAD

**Discussion by davidyew** [nautechsystems/nautilus_trader#781] [General]
Hi!
From Metatrader: https://www.metatrader4.com/en/trading-platform/help/service/history_center
It is possible to export from Historical Center 1 minute csv data which is in YYYY.MM.dd, HH:mm, open, high, low, close, volume.
May I know if there can be an example to show how backtesting with this data is possible?

**Comment by cjdsellers** [comment]
Hi @davidyew 

Thanks for the question. Its definitely possible and just requires you to define the correct data format, for that to then be wrangled into the appropriate Nautilus objects. Information on all of this can be found in the docs, you have given me a great idea to include a specific example for metatrader data - although it'll have to remain lower on the list of priorities for now

https://docs.nautilustrader.io/user_guide/loading_external_data.html
